cypress/integration/buy-shirt.spec.ts

```diff
@@ -0,0 +1,24 @@
+
+describe("Buy a t-shirt", () => {
+
+    it("then the t-shirt should be bought", () => {
+      cy.visit("http://automationpractice.com/")
+      cy.get("#block_top_menu > ul > li:nth-child(3) > a").click()
+      cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click()
+      cy.get("[style*=\"display: block;\"] .button-container > a").click()
+      cy.get(".cart_navigation span").click()
+  
+      cy.get("#email").type("aperdomobo@gmail.com")
+      cy.get("#passwd").type("WorkshopProtractor")
+  
+      cy.get("[name=SubmitLogin]").click()
+      cy.get("[name=processAddress]").click()
+      cy.get("[name=cgv]").click()
+      cy.get("[name=processCarrier]").click()
+      cy.get("[class=bankwire]").click()
+      cy.get("#cart_navigation > button").click()
+
+      cy.get("#center_column > div > p > strong")
+        .should("have.text", "Your order on My Store is complete.")
+    });
+  }); 
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon:
![image](https://user-images.githubusercontent.com/10209180/126734006-081807a8-bc9b-47eb-89f6-2d04a7dc18bd.png)